### PR TITLE
Update installation.md documentation

### PR DESCRIPTION
**Pull request by ballinette**
* remove RabbitMQ from prerequisites
* update doc about how to change external services settings

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -91,12 +90,24 @@ You may also start Kuzzle by using [PM2](https://github.com/Unitech/pm2):
 $ pm2 start app-start.js
 ```
 
-### Change RabbitMQ and Elasticsearch hosts
 
-If you are not running RabbitMQ and Elasticsearch on localhost, you can configure host and port:
+### Change external services hosts
+
+If you are some external service in another host than localhost, you can configure host and port
```
Comment (benoitvidis) [inline docs/installation.md]: If you are running some of the service(s) externally, you can configure their host and port using some environment variables:

Comment (ballinette) [inline docs/installation.md]: fixed

**Comment by benoitvidis** [comment]
+1

**Comment by scottinet** [comment]
+1

**Pull request merged**